Binary files kolourpaint/.git/index and kolourpaintn/.git/index differ
diff -Nur kolourpaint/kolourpaint.cpp kolourpaintn/kolourpaint.cpp
--- kolourpaint/kolourpaint.cpp	2015-12-17 14:27:46.000000000 +0800
+++ kolourpaintn/kolourpaint.cpp	2015-12-19 11:02:51.020885262 +0800
@@ -34,6 +34,7 @@
 
 #include <QApplication>
 #include <QCommandLineParser>
+#include <KLocalizedString>
 
 int main(int argc, char *argv [])
 {
@@ -77,6 +78,8 @@
 
   QApplication app(argc, argv);
 
+  KLocalizedString::setApplicationDomain("kolourpaint");
+
   QCommandLineParser cmdLine;
   KAboutData::setApplicationData(aboutData);
   cmdLine.addVersionOption();
